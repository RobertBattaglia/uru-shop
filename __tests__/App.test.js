// import React from 'react';
// import renderer from 'react-test-renderer';
// import { Provider } from 'react-redux';
// import App from '../client/src/components/App';
// import store from '../client/src/store';

// test('It renders App', () => {
//   const tree = renderer
//     .create(
//       <Provider store={store}>
//         <App />
//       </Provider>
//     )
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// });
