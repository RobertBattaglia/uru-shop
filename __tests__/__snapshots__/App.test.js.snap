// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders App 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <header>
    <header
      className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionSticky sc-htoDjs gLXJsq MuiAppBar-colorPrimary"
    >
      Greenfield Logo
    </header>
  </header>
  <main />
</div>
`;
